action-server/.vscode/settings.json

```diff
@@ -36,12 +35,11 @@
     "**/.venv": true,
     "**/.mypy_cache": true
   },
-  "python.testing.pytestArgs": [
-    "tests"
-  ],
   "python.testing.unittestArgs": [
     "-v",
     "-s",
     "./tests"
-  ]
+  ],
+  "python.linting.pylamaEnabled": true,
+  "python.pythonPath": ".venv/bin/python3"
```

Comment:
Is this path correct?

Reply:
Removed since vscode should be using the virtualenv and not a python path